tests/models/test_restore.py

```diff
@@ -360,9 +370,6 @@ def test_load_model_from_checkpoint(tmpdir, model_template):
     new_trainer = Trainer(**trainer_options)
     new_trainer.test(pretrained_model)
 
-    # test we have good test accuracy
-    tutils.assert_ok_model_acc(new_trainer, key='test_loss')
```

Comment:
@rohitgr7 Why did you remove this?

Reply:
it's not required here. all we need to ensure that weights are loaded correctly. else I had to update all the generic models added here.